src/experiments/sonar_cnn.py

```diff
@@ -0,0 +1,88 @@
+import os
+import random
+
+import numpy as np
+from sklearn.model_selection import KFold
+
+import utils.settings as settings
+from evaluation.metrics import accuracy
+from loader.Preprocessor import Preprocessor
+from loader.load_dataset import load_dataset
+from models.JensModel import JensModel
+from training.k_fold_test import k_fold_cross_validation_test
+from utils.filter_activities import filter_activities
+from utils.folder_operations import new_saved_experiment_folder
+
+# Init
+settings.init("sonar")
+random.seed(1678978086101)
+
+print(settings.ACTIVITIES)
+# Load & Prepare data
+
+# Pass a limit to only load a few recordings (for testing)
+recordings = load_dataset("/Users/franz/Projects/BP/new_data", limit=None)
+
+activities = len(settings.LABELS)
+
+recordings = Preprocessor().our_preprocess(recordings)
+
+# Filter activities & remove all recordings which have no matching activity
+
+# 8: Wagen schieben, 15: Haare kämmen, 29: Wäsche im Bett, 35: Essen austragen
+# 39: Getränke ausschenken, 52: Rollstuhl schieben, 55: Computerarbeit
+recordings = filter_activities(recordings, [8, 15, 29, 35, 39, 52, 55])
+recordings = list(filter(lambda rec: len(rec.sensor_frame) > 60, recordings))
```

Comment:
magic number, maybe define frequency instead? maybe we want to use 30hz data in the future

Reply:
Ideally it should work with just 
```python
recordings = list(filter(lambda rec: len(rec.sensor_frame) > 0, recordings))
```
(all not-empty recordings)